direct/data/datasets_config.py

```diff
@@ -19,8 +19,19 @@ class TransformsConfig(BaseConfig):
     crop_type: Optional[str] = "uniform"
     image_center_crop: bool = False
     padding_eps: float = 0.001
-    estimate_sensitivity_maps: bool = True
     estimate_body_coil_image: bool = False
+    random_rotation: bool = False
+    random_rotation_degrees: Tuple[int, ...] = (-90, 90)
+    random_rotation_probability: Optional[float] = 0.5
+    random_flip: bool = False
+    random_flip_type: Optional[str] = "random"
+    random_flip_probability: Optional[float] = 0.5
+    estimate_sensitivity_maps: bool = True
+    sensitivity_maps_type: str = "rss_estimate"
+    sensitivity_maps_espirit_threshold: Optional[float] = 0.05
+    sensitivity_maps_espirit_kernel_size: Optional[int] = 6
+    sensitivity_maps_espirit_crop: Optional[float] = 0.95
+    sensitivity_maps_espirit_max_iters: Optional[int] = 30
```

Comment:
Not a fan. Can't we have transforms a different config?| Like `transforms: TransformConfig`

Reply:
Will do in future PR